src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DocumentState.cs

```diff
@@ -265,8 +266,8 @@ private ImmutableArray<IDocumentSnapshot> GetImportsCore(ProjectSnapshot project
         return imports.ToImmutable();
     }
 
-    // See design notes on ProjectState.ComputedStateTracker.
-    private class ComputedStateTracker
+    // Internal, because we are temporarily sharing code with CohostDocumentSnapshot
```

Comment:
Is there a catch-all issue where we can add all of the changes we should put back when co-hosting is done?

Reply:
Rather than being put back, I would expect this whole file would be deleted in the end, because nothing references it. Not a bad idea to keep a list of these things though, as a head start.

Reply:
Yeah, the reason that I mentioned it was because I just happened upon a few internal methods with comments that said they were temporarily marked as virtual for VS Code. However, I suspect that the temporary need had expired.

Reply:
https://github.com/dotnet/razor/issues/9814